arrow_test.go

```diff
@@ -53,8 +54,9 @@ func TestArrowBigInt(t *testing.T) {
 }
 
 func TestArrowBigFloat(t *testing.T) {
-	db := openDB(t)
-	dbt := &DBTest{t, db}
+	conn := openConn(t)
```

Comment:
Do we guarantee that this test run always first to pool the connection?

Reply:
No, but it doesn't matter. Pool is managed by Go SQL and it works very well. I tried to run single test, multiple, all, and even tests in which connections were hang (using burp suite), and it worked as expected.